### ISSUE TITLE
Apt repository is gone

### ISSUE THREAD

**Issue by 1n17** [MyHush/hush#82]
### Describe the issue
The repository 'https://dl.bintray.com/myhush/hush hush Release' does not exist anymore.

#### If so, please list the steps to reproduce below:
1. sudo apt-get update
2. E: Failed to fetch https://dl.bintray.com/myhush/hush/dists/hush/main/binary-amd64/Packages  404  Not Found

### Expected behaviour
The repository should exist

### Actual behaviour + errors
The repository does not exist

**Comment by jebbgrenham** [comment]
Agreed, it's most definitely missing. All that remains is a windows binary.

**Comment by 1n17** [comment]
Hey @madbuda, here is another non-issue for you. Would you please take a look?

**Comment by madbuda** [comment]
Problem is when my ghithub access was removed it broke the org on bintray. It is all tied together... I am trying to find a new place to host the repo

**Comment by jebbgrenham** [comment]
So, how do we get the node binaries in the meantime?

**Comment by javierfd81** [comment]
hi, there is any way to get the binaries for linux in the meanwhile?

**Comment by nimbosa** [comment]
i was about to report this, but found #82

all that's left is an old Windows installer: https://dl.bintray.com/myhush/hush/:hush-1.0.12.msi

**Comment by nimbosa** [comment]
instruction @ MyHush/[INSTALL.md](https://github.com/MyHush/hush/blob/master/INSTALL.md) doesn't work
`Unable to locate package hush`
no deb found @ REPO: **dl.bintray.com/myhush**

can't install on Ubuntu:
```
echo 'deb https://dl.bintray.com/myhush/hush/ hush main' | sudo tee --append /etc/apt/sources.list.d/hush.list
apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
sudo apt-get update
sudo apt-get install hush
```

it's either we update the markdown OR reproduce the deb repo

**Comment by madbuda** [comment]
@nimbosa you are free to submit a PR with an updated install.md until we find a suitable repo to host this. I am working on a PPA, waiting on keys to be populated ATM
